packages/http-client/src/remote-admin-api.ts

```diff
@@ -88,8 +123,11 @@ export class RemoteAdminApi implements AdminApi {
         )}`,
       },
     })
-    return response.models.map((modelStreamIDString: string) => {
-      return StreamID.fromString(modelStreamIDString)
+    return response.map((resp) => {
+      return {
+        models: resp.models.map((model) => model.streamID.toString()),
+        indices: resp.models,
+      }
```

Comment:
This looks like a breaking change to the http api?

Reply:
I think it's non-breaking. I think the original response was `{'models':[]}` and it's now `{'models':[],'indices':[]}`

Reply:
The test has this as the expected response ```const GET_RESPONSE = {
  models: [MODEL.toString()],
}```

Reply:
The breaking change is actually elsewhere, where models should be an array of string StreamIDs, and indices should be the model indices

Reply:
Ok. Maybe we should do some cross testing for this PR?
* older client works with newer http api
* newer client works with older http api

Reply:
I think this is handled by the admin-api in stream-tests. This test makes http calls against ceramic, using remote admin api.

Reply:
the stream-tests tests all use the same version of the client and server.  We don't have any mixed-version compatibility tests currently